### PR TITLE
Allow pytest

### PR DESCRIPTION
**Pull request by dneise**
This PR assumes #82 is already merged. It's diff will be considerably smaller once #82 is merged.

I saw pytest was commented out in `.circleci/config.yml` and thought .. well pytest is pretty common nowadays.

So I renamed a file from `test_all.py` to `test_replays.py` and commented pytest in.

### CODE REVIEW

**Comment by dneise** [comment]
This also adds an important check, i.e.:

 **is this package really pip installable?**

File: .circleci/config.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ my-steps: &steps
   - run: sudo pip install -r requirements.txt
   - run: sudo pip install flake8 pytest
   - run: python --version ; pip --version ; pwd ; ls -l
+  - run: pip install --user .
```
Comment (cclauss) [inline .circleci/config.yml:8]: Not needed. CircleCI (like Travis CI) does this automatically.

Comment (dneise) [reply .circleci/config.yml:8]: Hmm .. I must admit, I do not know much about Circle .. but it failed before I added this.

Have a look here please: https://circleci.com/gh/ggtracker/sc2reader/259
```
Traceback:
test_replays/test_replays.py:21: in <module>
    import sc2reader
E   ModuleNotFoundError: No module named 'sc2reader'
```

Comment (cclauss) [reply .circleci/config.yml:8]: OK.  Go with what works.

Comment (dneise) [reply .circleci/config.yml:8]: ok

**Review by StoicLoofah — COMMENTED** [review]
I'm good with using pytest! If we want that to be our canonical test runner, can you also update our README accordingly? https://github.com/ggtracker/sc2reader#testing

**Comment by dneise** [comment]
Oh! A very very good point, I totally missed that!

**Comment by dneise** [comment]
@cclauss and @StoicLoofah can this be merged?

**Review by StoicLoofah — APPROVED** [review]
looks good!

**Pull request merged**